modules/api/php/endpoints/candidate/visit/image/format/raw.class.inc

```diff
@@ -140,13 +140,13 @@ class Raw extends Endpoint implements \LORIS\Middleware\ETagCalculator
 
         $callback = function () use ($mincpath, $fullpath) {
             return shell_exec(
-                "${mincpath}/bin/minctoraw -byte -unsigned -normalize $fullpath"
+                "{$mincpath}/bin/minctoraw -byte -unsigned -normalize $fullpath"
```

Comment:
I'm not sure what's simple or complex about `${foo}` vs `{$foo}` but phan was complaining that the former was deprecated and saying the latter should be used instead.

Reply:
For information:

https://www.php.net/manual/en/language.types.string.php

```
Simple syntax
If a dollar sign ($) is encountered, the parser will greedily take as many tokens as possible to form a valid variable name. Enclose the variable name in curly braces to explicitly specify the end of the name.

Complex (curly) syntax
Any scalar variable, array element or object property with a string representation can be included via this syntax. The expression is written the same way as it would appear outside the string, and then wrapped in { and }. Since { can not be escaped, this syntax will only be recognized when the $ immediately follows the {.
```

So, both worked in this context but the former is deprecated in 8.2. Probably because it can be achieved with the latter.